samples/sample_common/src/sample_utils.cpp

```diff
@@ -152,6 +174,23 @@ void CSmplYUVReader::Reset()
     }
 }
 
+mfxStatus CSmplYUVReader::ChangePositionInFile(const mfxFrameInfo& info, mfxU32 displayOrder)
```

Comment:
here is still issue with responsibilities. Reader class here doesn't care about surface at all. So this function should accept width and height as parameters

Reply:
Also let's name it more precisely, something like:
```
SkipNframesFromBeginning
```